### PR TITLE
chore: add LNDLTC Neutrino

### PR DESCRIPTION
**Pull request by michael1011**
Depends on https://github.com/ExchangeUnion/xud-docker/pull/356

### CODE REVIEW

File: Market Maker Guide.md [hunk 1]
```diff
@@ -172,15 +174,15 @@ mainnet > status
 │ xud       │ Waiting for sync                               │
 └───────────┴────────────────────────────────────────────────┘
 ```
-The full setup starts syncing fast and gets slower towards the end. Bitcoind/Litecoind should finish syncing within 12h, whereas geth will need about 3 full days. The light setup should be ready within 2-3h (there is no working light mode for LTC yet). A Pi4 needs about twice that long.
+The full setup starts syncing fast and gets slower towards the end. Bitcoind/Litecoind should finish syncing within 12h, whereas geth will need about 3 full days. The light setup should be ready within one hour. A Pi4 needs about twice that long.
```
Comment (sangaman) [inline Market Maker Guide.md:177]: Not exactly related to this PR, but the last line here "A Pi4 needs about twice that long" isn't clear whether it refers to the full node setup, the light setup, or both.

**Review by sangaman — APPROVED** [review]
LGTM

File: Market Maker Guide.md (hunk 1 above)
Comment (michael1011) [reply Market Maker Guide.md:177]: Good point. @kilrau what was the initial thought behind this?

Comment (kilrau) [reply Market Maker Guide.md:177]: Initial thought was that it's valid for Light+Full. But how about I set up a fresh all-light setup on my arm64 Pi4 & on my fast amd64 laptop right now and check how long it takes and circle back.

Comment (kilrau) [reply Market Maker Guide.md:177]: I'll fix that up in a separate PR though and the docs need to be updated again anyways after https://github.com/ExchangeUnion/xud-docker/pull/343 is merged.

**Review by kilrau — APPROVED** [review]

**Pull request merged**